protocol/x/clob/keeper/deleveraging.go

```diff
@@ -162,21 +162,26 @@ func (k Keeper) OffsetSubaccountPerpetualPosition(
 	)
 
 	numSubaccountsIterated := 0
+	numSubaccountsWithNonOverlappingBankruptcyPrices := 0
+	numSubaccountsWithNoOpenPositionOnOppositeSide := 0
```

Comment:
Suggestion, up to you: for clarity why not use the `uint` type?

Reply:
I think it's pretty clear that it's a counter (non negative) by their names. they get converted to float32 anyways, so doesn't really matter

updated since it's an easy change